### PR TITLE
[Stable Diffusion] Set traffic separately for CLI examples

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
# Description
[Stable Diffusion] Set traffic separately for CLI examples

Made changes to the removed "/batch_data" of  datapath  in batch deployment and  online endpoint  in cli's files

Faced the token issue while deploying model from registry to workspace in online endpoint  cli files modified the code  as below

# Deploy model from registry to endpoint in workspace
az ml online-deployment create --file deploy-online.yaml $workspace_info --set \
endpoint_name=$endpoint_name model=azureml://registries/$registry_name/models/$model_name/versions/$model_version \
instance_type=$deployment_sku || {
echo "deployment create failed"; exit 1;
}
yaml_file="deploy-online.yaml"
get_yaml_value() {
    grep "$1:" "$yaml_file" | awk '{print $2}' | sed 's/[",]//g'
}
traffic=$(get_yaml_value "name")

az ml online-endpoint update $workspace_info --name=$endpoint_name --traffic="$traffic=100"

# Checklist

- [x] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Comment by KonakanchiSwathi** [comment]
@rjaincc @MadhuM02 Can you please review Thanks in Advance

File: cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh [hunk 1]
```diff
@@ -49,10 +49,10 @@ az ml online-endpoint create --name $endpoint_name $workspace_info  || {
...
 
 # Deploy model from registry to endpoint in workspace
-az ml online-deployment create --file deploy-online.yaml $workspace_info --all-traffic --set \
-  endpoint_name=$endpoint_name model=azureml://registries/$registry_name/models/$model_name/versions/$model_version \
-  instance_type=$deployment_sku || {
-    echo "deployment create failed"; exit 1;
+az ml online-deployment create --file deploy-online.yaml $workspace_info --set \
+endpoint_name=$endpoint_name model=azureml://registries/$registry_name/models/$model_name/versions/$model_version \
+instance_type=$deployment_sku || {
+echo "deployment create failed"; exit 1;
```
Comment (rjaincc) [inline cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh:55]: Traffic needs to be set.

Comment (KonakanchiSwathi) [reply cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh:55]: Traffic value is set

File: cli/foundation-models/system/inference/text-to-image/text-to-image-batch-endpoint.sh [hunk 2]
```diff
@@ -52,7 +52,7 @@ else
 fi
 
 # 4. Submit a sample request to endpoint
-data_path="./text_to_image_batch_data/batch_data"
+data_path="./text_to_image_batch_data"
```
Comment (rjaincc) [inline cli/foundation-models/system/inference/text-to-image/text-to-image-batch-endpoint.sh:55]: is this required? Have we verified it?

**Review by rjaincc — APPROVED** [review]
Approving with comments. 
- Please update the description of the PR to align with the changes in the PR. 
- Please check if the changes in the batch example are required?

File: cli/foundation-models/system/inference/text-to-image/text-to-image-batch-endpoint.sh (hunk 2 above)
Comment (KonakanchiSwathi) [reply cli/foundation-models/system/inference/text-to-image/text-to-image-batch-endpoint.sh:55]: yes As per discussion with Madhu updated it @MadhuM02 please feel free to provide your comments

File: cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh [hunk 3]
```diff
@@ -49,12 +49,18 @@ az ml online-endpoint create --name $endpoint_name $workspace_info  || {
...
 }
+yaml_file="deploy-online.yaml"
+get_yaml_value() {
+    grep "$1:" "$yaml_file" | awk '{print $2}' | sed 's/[",]//g'
+}
+traffic=$(get_yaml_value "name")
```
Comment (MadhuM02) [inline cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh:61]: change this "traffic" to deployment-name

Comment (KonakanchiSwathi) [reply cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh:61]: ok

**Review by rjaincc — APPROVED** [review]

**Review by MadhuM02 — APPROVED** [review]

File: cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh [hunk 4]
```diff
@@ -55,14 +55,22 @@ max_concurrent_request=2  # the maximum number of concurrent requests supported
 # If you are using a larger SKU, please increase this value to get the maximum performance.
 
 # Deploy model from registry to endpoint in workspace
+
 az ml online-deployment create --file deploy-online.yaml $workspace_info --all-traffic --set \
```
Comment (vadthyavath) [inline cli/foundation-models/system/inference/text-to-image/text-to-image-online-endpoint.sh:59]: this step runs into " (InvalidAuthenticationTokenTenant) The access token is from the wrong issuer"

 issue

**Review by vadthyavath — CHANGES_REQUESTED** [review]

**Pull request closed without merging**